### DISCUSSION TITLE
Error while running the project in macOS intel

### DISCUSSION THREAD

**Discussion by Hellstellar** [Sunbird-Lern/Community#85] [Issues]
While setting up the project all the steps were followed. When I run `mvn play2:run` I am getting the following error:

`java.lang.NoClassDefFoundError: com/google/code/play2/plugin/MavenPlay2BuilderRunnable`

The above error is coming in java version: 11.0.2

A slightly different error was coming when I was using java version: 11.0.18:

`Failed to execute goal com.google.code.play2-maven-plugin:play2-maven-plugin:1.0.0-rc5:run (default-cli) on project lms-service: ?: ExceptionInInitializerError: NullPointerException`

Maven version is 3.9.0

**Comment by rhwarrier** [comment]
@reshmi-nair  @BharathwajShankar  can you pls post an update here in terms of the issue/ what was tried out, so that other users can also benefit from it?

**Comment by Manjunath375** [comment]
https://github.com/orgs/Sunbird-Lern/discussions/155 Facing same issue any solution for this error do share it

**Reply by Jayaprakash8887** [reply]
Issue in MacOs Intel chipset can be resolved by switching between java openJdk and oracle jdk. However, we have not yet been able to resolve issues for M1 and M2 chipset.

**Reply by AmiableAnil** [reply]
@Manjunath375 can we connect at 2:00PM today? Please let me know, so that I'll share the link here.
